code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+		icon_state = "bridge"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	brow_left
+		name = "Brow Left"
+		icon_state = "brow_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	brow_right
+		name = "Brow Right"
+		icon_state = "brow_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	cheekspot_left
+		name = "Cheek Spot (Left Cheek)"
+		icon_state = "cheekspot_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	cheekspot_right
+		name = "Cheek Spot (Right Cheek)"
+		icon_state = "cheekspot_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	cheshire_left
+		name = "Cheshire (Left Cheek)"
+		icon_state = "cheshire_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	cheshire_right
+		name = "Cheshire (Right Cheek)"
+		icon_state = "cheshire_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	crow_left
+		name = "Crow Mark (Left Eye)"
+		icon_state = "crow_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	crow_right
+		name = "Crow Mark (Right Eye)"
+		icon_state = "crow_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	ear_left
+		name = "Ear Cover (Left)"
+		icon_state = "ear_l"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	ear_right
+		name = "Ear Cover (Right)"
+		icon_state = "ear_r"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	eyestripe
+		name = "Eye Stripe"
+		icon_state = "eyestripe"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyecorner_left
+		name = "Eye Corner Left"
+		icon_state = "eyecorner_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyecorner_right
+		name = "Eye Corner Right"
+		icon_state = "eyecorner_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyelash_left
+		name = "Eyelash Left"
+		icon_state = "eyelash_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	eyelash_right
+		name = "Eyelash Right"
+		icon_state = "eyelash_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	fullfacepaint
```

Comment:
This needs to be removed, it makes zero sense in the role play environment of the station

Reply:
Can you actually explain why.

Reply:
Full face paint makes no sense in a working enviroment